### ISSUE TITLE
*awx/main/migrations uid:gid should be awx:awx ? 

### ISSUE THREAD

**Issue by tjyang** [MrMEEE/awx-build#11]
* I try to follow the upgrade procedure encounter following issue
```
[root@ansible01 ~]# sudo -u awx /opt/awx/bin/awx-manage makemigrations
Migrations for 'main':
  /opt/awx/embedded/lib/python2.7/site-packages/awx/main/migrations/0014_auto_20180410_1811.py
    - Alter field created_by on credential
    - Alter field modified_by on credential
    - Alter field created_by on credentialtype
    - Alter field modified_by on credentialtype
    - Alter field created_by on custominventoryscript
    - Alter field modified_by on custominventoryscript
    - Alter field created_by on group
    - Alter field modified_by on group
    - Alter field created_by on host
    - Alter field modified_by on host
    - Alter field created_by on inventory
    - Alter field modified_by on inventory
    - Alter field created_by on label
    - Alter field modified_by on label
    - Alter field created_by on notificationtemplate
    - Alter field modified_by on notificationtemplate
    - Alter field created_by on organization
    - Alter field modified_by on organization
    - Alter field created_by on schedule
    - Alter field modified_by on schedule
    - Alter field created_by on team
    - Alter field modified_by on team
    - Alter field created_by on unifiedjob
    - Alter field modified_by on unifiedjob
    - Alter field created_by on unifiedjobtemplate
    - Alter field modified_by on unifiedjobtemplate
Traceback (most recent call last):
  File "/opt/awx/bin/awx-manage", line 11, in <module>
    sys.exit(manage())
  File "/opt/awx/embedded/lib/python2.7/site-packages/awx/__init__.py", line 109, in manage
    execute_from_command_line(sys.argv)
  File "/opt/awx/embedded/lib64/python2.7/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/opt/awx/embedded/lib64/python2.7/site-packages/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/opt/awx/embedded/lib64/python2.7/site-packages/django/core/management/base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/opt/awx/embedded/lib64/python2.7/site-packages/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/opt/awx/embedded/lib64/python2.7/site-packages/django/core/management/commands/makemigrations.py", line 193, in handle
    self.write_migration_files(changes)
  File "/opt/awx/embedded/lib64/python2.7/site-packages/django/core/management/commands/makemigrations.py", line 232, in write_migration_files
    with io.open(writer.path, "w", encoding='utf-8') as fh:
IOError: [Errno 13] Permission denied: '/opt/awx/embedded/lib/python2.7/site-packages/awx/main/migrations/0014_auto_20180410_1811.py'
[root@ansible01 ~]# 
```

**Comment by tjyang** [comment]
This permission issue is fixed at 1.0.5 version rpm.

**Comment by MrMEEE** [comment]
Great.. thanks for testing :)
